src/Framework/MSBuildEventSource.cs

```diff
@@ -80,9 +80,9 @@ public void RarComputeClosureStart()
         }
 
         [Event(8, Keywords = Keywords.All)]
-        public void RarComputeClosureStop()
+        public void RarComputeClosureStop(int referencesResolved)
         {
-            WriteEvent(8);
+            WriteEvent(8, referencesResolved);
```

Comment:
What else needs to be modified to pick up this int?

Reply:
I'm not sure I follow?

Reply:
I forget exactly how these events work. My assumption is by adding a parameter to this, you need to update something elsewhere to pick it up. Similar to a resource string.

Reply:
I don't think it's possible to localize these. The ETW traces take the name they display in PerfView directly from the argument name, and we can't change that per locality (currently, although I don't see why that couldn't be something to do if we ever run out of things to work on).

Reply:
Ah gotcha, I didn't realize ETW does that automatically.